server/api/templates/docs/pd.swagger.json

```diff
@@ -0,0 +1,809 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "description": "REST APIs provided by Placement Driver that is used to manage and schedule the TiKV cluster.",
```

Comment:
The REST APIs provided by the Placement Driver that can be used to manage and schedule the TiKV cluster.